cmd/indexer/internal/datastore/pq/mappings_test.go

```diff
@@ -0,0 +1,73 @@
+package pq
+
+import (
+	"testing"
+
+	"github.com/mailchain/mailchain/internal/protocols"
+	"github.com/mailchain/mailchain/internal/protocols/ethereum"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetProtocolNetworkUint8(t *testing.T) {
+	assert := assert.New(t)
+	type args struct {
+		protocol string
+		network  string
+	}
+	type result struct {
+		protocol uint8
+		network  uint8
+		wantErr  bool
+	}
+	tests := []struct {
+		name   string
+		args   args
+		result result
+	}{
+		{
+			"success",
+			args{
+				protocols.Ethereum,
+				ethereum.Mainnet,
+			},
+			result{
+				1,
+				1,
+				false,
+			},
+		},
+		{
+			"protocol unknown",
+			args{
+				"unknown",
+				ethereum.Mainnet,
+			},
+			result{
+				0,
+				0,
+				true,
+			},
+		},
+		{
+			"network unknown",
```

Comment:
I found for errors its good to put a prefix e.g. `err`. When tests fail, test runners can render the whitespace with escape characters making it a bit more difficult to find the failing test case.
I would suggest a name like `err-network-unknown`